Document/Structure/ArticleBridge.php

```diff
@@ -30,14 +30,26 @@ class ArticleBridge extends StructureBridge implements RoutableStructureInterfac
 
     public function getView(): string
     {
-        return $this->structure->getView();
+        /** @var string $view */
+        $view = $this->structure->getView();
+
+        return $view;
```

Comment:
No additional variable required: https://phpstan.org/r/37b31eef-6e58-4b1a-8134-6692c7830d33

```suggestion
        /** @var string */
        return $this->structure->getView();;
```